docs-content/general/4_company/1_values.md

```diff
@@ -1,32 +1,29 @@
 ---
-title: Mission & Values
+title: Values
 slug: values
 createdAt: 2022-04-01T20:28:14.000Z
 updatedAt: 2022-04-15T02:07:22.000Z
 ---
 
-Our mission is to support developers (like you) to ship with confidence. We do this by giving you the tools you need to **uncover, resolve, and prevent** issues in your web app.
+## Overview
 
-With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings. 
-
-## We build in public.
+This doc acts as a reference for our values at highlight.io, which includes how we work and we build things. It acts a way for us to understand how we should operate and what the "startup mentality" means to us. If you'd like to learn more about our product philosophy (which is more related to our product) see [this doc](./product-philosophy.md).
```

Comment:
* The first two sentences are both "acts as a..." can be consolidated into one sentence
* Second setence "It acts a way for us" -> "It acts **as** a way for us"
* "this doc does this" is overly wordy for no reason -- I'm reading this doc so we don't need to name the doc specifically.
* Same with "product philosophy (which is more related to our product)" -- of course "product philosophy" is related to our product 
* Mentioning the "startup mentality"